src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php

```diff
@@ -84,13 +85,14 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $documentRoot = $input->getOption('docroot');
+        $outputStyle = new SymfonyStyle($input, $output);
```

Comment:
As I said in https://github.com/symfony/symfony/pull/14132#discussion_r27778413, I don't see why we should absolutly name this variable differently, since it implements `OutputInterface`, only wraps the standard $output and behaves the same.
IMO, we should name it the same as the standard output as soon as we can.